tools/manifest/tests/test_manifest.py

```diff
@@ -353,13 +353,13 @@ def test_no_update():
     assert list(m) == [("testharness", test1.path, {test1}),
                        ("testharness", test2.path, {test2})]
 
-    s1_1 = SourceFileWithTest("test1", "1"*40, item.TestharnessTest)
+    s1_1 = SourceFileWithTest("test1", "1"*40, item.ManualTest)
```

Comment:
Let's rename the test, given this is now a fundamentally different test (and I think both are worth having, on the whole); this test was "no update" from the point of view of the manifest items, but now it has a notable change.

Reply:
I guess I simply didn't understand the original test. What was it testing and what did "no update" mean?

The most interesting part is the next: it used to be just `m.update([(s1, True), (s2.rel_path, False)])` and the newly created `s1_1` wasn't used. It seems like it was simply testing the handling of the (rel_path, False) type argument.

Reply:
So the original goal of the test was that it produced an identical set of tests before/after the update. I could've screwed up when I originally wrote it, though. 🙂